.travis.yml

```diff
@@ -20,12 +18,8 @@ matrix:
       env: TOXENV=py38
       dist: xenial
       sudo: true
-    - python: pypy
-      env: TOXENV=pypy
     - python: pypy3
       env: TOXENV=pypy3
-    - python: 3.5
-      env: TOXENV=py2-cover,py3-cover,coverage
```

Comment:
Please restore coverage.

Reply:
ups, I am sorry for late response :( I somehow overlooked your request, sorry for that. Anyway, this request is superseded already.